examples/demo.css

```diff
@@ -270,10 +268,22 @@ a.name-header{
 .col-md-4{
   padding-left: 30px;
 }
+/*
 .panel-heading{
   height:45px;
 }
 .panel-footer{
   height: 45px;
 }
-*/
+#gif{
```

Comment:
Why is this commented?

Reply:
@jywarren did it in the bootstrap panel.